UPGRADE-1.9.md

```diff
@@ -135,3 +154,6 @@
 1. We've replaced deprecated Symfony Translator API with the new one.
    
     Replace `Symfony\Component\Translation\TranslatorInterface` with `Symfony\Contracts\Translation\TranslatorInterface` in your codebase.
+
+1. We've changed `/new-api` prefix to `/api/v2`. If you're using API, replace api request url to `/api/v2`.
+   Old api is hardcoded to `/api/v1`.
```

Comment:
```suggestion
   Admin Api was hardcoded to `/api/v1`(instead of /api/v{version}).
```

Reply:
If I see correctly, in Admin API there is still `/api/v{version}`